File: C_Programming/Study/Q2_Assignment29.c
///////////////////////////////////////////////////////////////////////////////////////////////////////////////////
//
// Function Name :  ChkCapital
// Description:     Assignment_29: Q.2 :Accept a character from user & check whether it is Capital or not(A-Z).                                       
// Input : F        
// Output: True
// Input:  d 
// Output: False       
//          
// Date:            15/06/2026
// Author:          Shital Ajit Nikam   
//
/////////////////////////////////////////////////////////////////////////////////////////////////////////////////

#include<stdio.h>
#define TRUE 1
#define FALSE 0

typedef int BOOL;
BOOL ChkCapital(char ch)
{
    BOOL bFlag = FALSE;

    if(ch >= 'A' && ch <= 'Z')
    {
        bFlag = TRUE;
    }
    else
    {
        return bFlag;
    }

    return bFlag;

}

int main()
{
    char cValue = '\0';
    BOOL bRet = FALSE;

    printf("Enter the character\n");
    scanf("%c",&cValue);

    bRet = ChkCapital(cValue);

    if(bRet == TRUE)
    {
        printf("It is Capital character\n");
    }
    else
    {
        printf("It is not Capital character\n");
    }

    return 0;
}

File: Python_Programming/Competative/Q2_Assignment29.py
#///////////////////////////////////////////////////////////////////////////////////////////////////////////////////
#        
# Description: Assignment_29: Q.1: write a program which accept name of file from user, open that File & Displays  
#                                      entire content of file on console.
# Input  :  Demo.txt       
# Output :  Display content of Demo.txt                                    
#                                     
# Date:            19/07/2026
# Author:          Shital Ajit Nikam   
#
#///////////////////////////////////////////////////////////////////////////////////////////////////////////////////

import os

#///////////////////////////////////////////////////////////////////////////////////////////////////////////////////
# Function Name: Display: Display file content on console.
#///////////////////////////////////////////////////////////////////////////////////////////////////////////////////


def Display(FileName):
    try:
        fobj = open(FileName,"r")                  # read mode
        data = fobj.read()
        print(data)         
    
    
    except FileNotFoundError :
        print("Unable to open file, file does not exist.")
        return None            
                

def main():        
        
        Border = "-"*60
        print(Border)         
        
        fname = input("Enter the File name :")
        
        print(Border)
        print("File contents are : ")
        
        Display(fname)                           
                   
        print(Border)                    
    
        
if __name__ == "__main__" :
    main()